__tests__/screens/Details/TrackList/index.test.js

```diff
@@ -2,14 +2,14 @@ import React from 'react';
 import { render, screen, waitFor } from '@testing-library/react-native';
 import axios from 'axios';
 import { SafeAreaProvider } from 'react-native-safe-area-context';
-import TrackList from '../../../../src/screens/Details/TrackList';
-import whoSampledPayload, {
-  emptyPayload,
-} from '../../../fixtures/api/whosampled/sample-info.0';
+import SampleList from '../../../../src/screens/CurrentTrack/TrackList';
```

Comment:
Check the overall coverage of these tests and work through bumping it to 80% minimum